ansible/roles/cassandra-db-update/tasks/main.yml

```diff
@@ -13,11 +13,15 @@
   with_items:
     - "data.cql"
     - "dialcode.cql"
+  delegate_to: "{{ groups['cassandra'][0] }}"
+  run_once: true
 
   
 - name: run cql 
   become: yes
   shell: 'cqlsh -f /tmp/lp-{{item}}'
   with_items:
     - "data.cql"
-    - "dialcode.cql"
\ No newline at end of file
+    - "dialcode.cql"
+  delegate_to: "{{ groups['cassandra'][0] }}"
```

Comment:
I think we can remove the deligate to, as it doesn't make any sense to club with run_once

Reply:
Done.